frame/election-provider-support/src/lib.rs

```diff
@@ -338,10 +338,11 @@ pub trait SortedListProvider<AccountId> {
 		weight_of: Box<dyn Fn(&AccountId) -> VoteWeight>,
 	) -> u32;
 
-	/// Remove `maybe_count` number of items from the list. Returns the number of items actually
-	/// removed. WARNING: removes all items if `maybe_count` is `None`, which should never be done
-	/// in production settings because it can lead to an unbounded amount of storage accesses.
-	fn clear(maybe_count: Option<u32>) -> u32;
+	/// Remove all items from the list.
+	///
+	/// WARNING: should never be called in production settings because it can lead to an unbounded
+	/// amount of storage accesses.
```

Comment:
```suggestion
	/// ### WARNING
	/// This function should never be called in production settings because it can lead to an
	/// unbounded amount of storage accesses.
```